.golangci.yaml

```diff
@@ -5,6 +5,9 @@ linters:
     - gofmt
     - whitespace
     - govet
+    - misspell
+    - forcetypeassert
+    - gci
```

Comment:
Aded this three to catch hints raised by the pipeline (I would add more as I get more linter hints)